4.analyze_data/notebooks/linear_model/nbconverted/0.fit_linear_model.py

```diff
@@ -52,8 +54,8 @@
 cell_count_df.columns = ["Metadata_Well", "Metadata_cell_count_per_well"]
 cp_df = cell_count_df.merge(cp_df, on=["Metadata_Well"])
 
-# Clean the dose column to extract numeric value
-cp_df = cp_df.assign(Metadata_dose_numeric=cp_df.Metadata_dose.str.strip("uM").astype(float))
+# # Clean the dose column to extract numeric value
+# cp_df = cp_df.assign(Metadata_dose_numeric=cp_df.Metadata_dose.str.strip("uM").astype(float))
```

Comment:
If this code is not used can the code and comment above be removed?

Reply:
Hmm, it is needed depending on the plate being analyzed. I will clarify this in the code comment, thank you for bringing this up!